uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -265,12 +274,17 @@ async def asgi_send(self, message: "ASGISendEvent") -> None:
                 self.accepted_subprotocol = cast(
                     Optional[Subprotocol], message.get("subprotocol")
                 )
-                if "headers" in message:
-                    self.extra_headers.extend(
-                        # ASGI spec requires bytes
-                        # But for compatibility we need to convert it to strings
-                        (name.decode("latin-1"), value.decode("latin-1"))
-                        for name, value in message["headers"]
+                headers = list(message.get("headers", []))
+                for name, value in headers:
+                    if name.lower() in [b"server"]:
+                        continue
```

Comment:
Websockets doesn't allow multiple `Server` headers. This was added to prevent it from failing. 

Alternatively we can raise a `RuntimeError` as well.